.github/workflows/testing-new-gateway.yml

```diff
@@ -73,8 +73,11 @@ jobs:
 
           # Disabling the compute api for now
           # Check https://github.com/nevermined-io/sdk-js/issues/33
-
-          ./start_nevermined.sh --latest ${{ matrix.network }} --no-graph --no-gateway --opengsn 2>&1 | tee nevermined_tools.txt &
+          
+          # TODO: Use the default Node when merged into master
+          export KEEPER_VERSION=latest
+          export NODE_VERSION=latest
```

Comment:
aren't we already using latest by default?
also I don't think we need to `NODE_VERSION` since the nevermined tools is started with `--no-node`